tests/snuba/api/endpoints/test_organization_events_measurements_histogram.py

```diff
@@ -0,0 +1,413 @@
+        query = {
+            "project": [self.project.id],
+            "numBuckets": 10,
+        }
+
+        response = self.do_request(query)
+        assert response.status_code == 400
+        assert "Missing value for measurements." in response.data["detail"]
+
+    def test_bad_params_too_many_measurements(self):
+        query = {
+            "project": [self.project.id],
+            "measurement": ["foo", "bar", "baz", "qux", "quux"],
+            "numBuckets": 10,
+            "min": 0,
+            "max": 100,
+            "precision": 0,
+        }
+
+        response = self.do_request(query)
+        assert response.status_code == 400
+        assert "Too many measurements specified, maximum allowed is 4." in response.data["detail"]
+
+    def test_bad_params_missing_numBuckets(self):
+        query = {
+            "project": [self.project.id],
+            "measurement": ["foo", "bar"],
+        }
+        response = self.do_request(query)
+        assert response.status_code == 400
+        assert "Missing value for numBuckets." in response.data["detail"]
+
+    def test_bad_params_invalid_numBuckets(self):
+        query = {
+            "project": [self.project.id],
+            "measurement": ["foo", "bar"],
+            "numBuckets": "baz",
+        }
+        response = self.do_request(query)
+        assert response.status_code == 400
+        assert (
+            "Invalid value for numBuckets. Expected to be at least 1 got baz."
+            in response.data["detail"]
+        )
+
+    def test_bad_params_invalid_negative_numBuckets(self):
+        query = {
+            "project": [self.project.id],
+            "measurement": ["foo", "bar"],
+            "numBuckets": -1,
+        }
+        response = self.do_request(query)
+        assert response.status_code == 400
+        assert (
+            "Invalid value for numBuckets. Expected to be at least 1 got -1"
+            in response.data["detail"]
+        )
+
+    def test_bad_params_invalid_precision(self):
+        query = {
+            "project": [self.project.id],
+            "measurement": ["foo", "bar"],
+            "numBuckets": 10,
+            "precision": -1,
+        }
+
+        response = self.do_request(query)
+        assert response.status_code == 400
+        assert (
+            "Invalid value for precision. Expected to be between 0 and 4 got -1"
+            in response.data["detail"]
+        )
+
+    def test_histogram_empty(self):
+        specs = [(i, i + 1, [("foo", 0), ("bar", 0)]) for i in range(5)]
+
+        query = {
+            "project": [self.project.id],
+            "measurement": ["foo", "bar"],
+            "numBuckets": 5,
+        }
+
+        response = self.do_request(query)
+        assert response.status_code == 200
+        assert response.data["data"] == self.as_response_data(specs)
+
+    def test_histogram_simple(self):
+        # range is [0, 5), so it is divided into 5 buckets of width 1
```

Comment:
```suggestion
        # range is [0, 5], so it is divided into 5 buckets of width 1
```

Reply:
This was intentional as 5 is exclusive. Should I be more explicit about this?

Reply:
No, I thought it might be that but it could have also been a typo.